nextflow_schema.json

```diff
@@ -10,24 +10,53 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir"],
+            "required": [
+                "input",
+                "outdir"
+            ],
             "properties": {
                 "input": {
-                    "type": "string",
-                    "format": "file-path",
-                    "exists": true,
-                    "mimetype": "text/csv",
-                    "pattern": "^\\S+\\.csv$",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/createpanelofnormals/usage#samplesheet-input).",
-                    "fa_icon": "fas fa-file-csv"
+                    "fa_icon": "fas fa-file-csv",
+                    "schema": "assets/schema_input.json",
+                    "anyOf": [
+                        {
+                            "type": "string",
+                            "format": "file-path",
+                            "exists": true,
+                            "mimetype": "text/csv",
+                            "pattern": "^\\S+\\.csv$"
+                        },
+                        {
+                            "type": "boolean",
+                            "enum": [
+                                "false"
+                            ]
+                        }
+                    ]
```

Comment:
Just curious. That snippet is new to me. That basically says `--input` may be false or a csv file, right? Is that related to nf-validation (because you use also schema above) or is that independent?

Reply:
ah good catch, i copied it from sarek. there we basically say, either `--input`must be provided with a `csv`file or you can not provide, and then instead we do something on the pipeline level to determine the input. It is to allow several forms of how you can provide the input.

Reply:
I see, thanks.

Reply:
ah ok, so I should always have that in there?

Reply:
if input is always a samplesheet path you can remove the `anyOf` section, this is only for Sarek special cases :)

Reply:
> ah ok, so I should always have that in there?

I understood like its not needed because `--input false` isnt evaluated anyway? This is also the experience I made.

Reply:
Actually, not needed anywhere as you say!

Reply:
ah yes that makes a lot more sense :D